packages/gp2-components/src/templates/__tests__/OnboardingAdditionalDetails.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { gp2 } from '@asap-hub/fixtures';
+import { render, screen } from '@testing-library/react';
+import OnboardingAdditionalDetails from '../OnboardingAdditionalDetails';
+
+const { createUserResponse } = gp2;
+
+describe('OnboardingAdditionalDetails', () => {
+  const defaultProps = {
+    ...createUserResponse(),
+    editQuestionsHref: '',
+    editFundingStreamsHref: '',
+    editContributingCohortsHref: '',
+    editExternalProfilesHref: '',
+  };
+
+  it('renders the page description', () => {
```

Comment:
I often forget to to it, but since all test look the same this could a `it.each`